sensor/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/sensor_residual.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MARKER_RADAR_LIDAR_CALIBRATOR__SENSOR_RESIDUAL_HPP_
+#define MARKER_RADAR_LIDAR_CALIBRATOR__SENSOR_RESIDUAL_HPP_
+
+#include <Eigen/Dense>
+
+namespace marker_radar_lidar_calibrator
+{
+
+struct SensorResidual
+{
+  SensorResidual(const Eigen::Vector4d & radar_point, Eigen::Vector4d & lidar_point)
+  : radar_point_(radar_point), lidar_point_(lidar_point)
+  {
+  }
+
+  template <class T>
+  bool operator()(T const * const params, T * s_residuals) const
+  {
+    // parameters: x, y, z, pitch, yaw.
+    Eigen::Matrix<T, 4, 4> s_transformation = Eigen::Matrix<T, 4, 4>::Identity(4, 4);
```

Comment:
s_transformation -> transformation_matrix?

Reply:
Thanks! fix in https://github.com/tier4/CalibrationTools/pull/161/commits/6f8296d4327d1f76883c2ae140a7a4187534c237

Reply:
*Fixed in

Reply:
The change does not appear in the commit you mention.
Please separate the commits and make sure you paste the appropriate one

Reply:
Sorry, I paste the wrong link.
correct one is here https://github.com/tier4/CalibrationTools/pull/161/commits/f2c0e02c43e460b52bd512e6ea1fc8785d92cad2